tests/integration/TechRecordsFunction.intTest.ts

```diff
@@ -166,14 +158,22 @@ describe("postTechRecords", () => {
           techRecord: techRecord.techRecord,
         };
 
+        const expectedResponse = {
+          vin: techRecord.vin,
+          primaryVrm: payload.primaryVrm,
+          techRecord: techRecord.techRecord,
+        };
+
         await LambdaTester(PostTechRecordsFunction)
           .event({
             path: "/vehicles",
             body: payload,
           })
           .expectResolve((result: any) => {
+            expectedResponse.techRecord[0].createdAt = JSON.parse(result.body).techRecord[0].createdAt;
+
             expect(result.statusCode).toEqual(201);
-            expect(result.body).toEqual('"Technical Record created"');
+            expect(JSON.parse(result.body)).toMatchObject(expectedResponse);
           });
```

Comment:
```suggestion
            expect(JSON.parse(result.body)).toMatchObject(expectedResponse);
            expect(JSON.parse(result.body).systemNumber).toBeDefined();
          });
```
More of a nice to have suggestion, since we do ideally want the systemNumber in the response (and it should be there!)